host/platform/Windows/capture/DXGI/src/d3d12.c

```diff
@@ -120,8 +121,8 @@ static bool d3d12_create(struct DXGIInterface * intf)
     return false;
   }
 
-  this->copySleep = option_get_int("dxgi", "d3d12CopySleep");
-  DEBUG_INFO("Sleep before copy : %d ms", this->copySleep);
+  this->copySleep = option_get_float("dxgi", "d3d12CopySleep");
```

Comment:
`d3d12CopySleep` is declared as `OPTION_TYPE_INT`, so this would just generate garbage.